packages/nocodb/src/lib/meta/api/dataApis/bulkDataAliasApis.ts

```diff
@@ -6,70 +6,61 @@ import ncMetaAclMw from '../../helpers/ncMetaAclMw';
 import { getViewAndModelFromRequestByAliasOrId } from './helpers';
 import apiMetrics from '../../helpers/apiMetrics';
 
-async function bulkDataInsert(req: Request, res: Response) {
+async function getModelAndBase(req: Request) {
```

Comment:
Revise the function name as it also return View as well, not just model and base.